### ISSUE TITLE
[BUG] Dosen't show device activation code

### ISSUE THREAD

**Issue by BenjaminHoegh** [gobijan/nova-copilot-lsp#5]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

<img width="643" alt="Skærmbillede 2023-05-23 kl  19 00 01" src="https://github.com/gobijan/nova-copilot-lsp/assets/22152591/8a9d7770-1ff5-4325-a841-d1d2690a3c3b">

**Comment by devinbaeten** [comment]
Also experiencing this in Nova 11.0

**Comment by Westindies972** [comment]
Same issue for me in Nova 11

**Comment by Westindies972** [comment]
Thanks a lot, I just tried, it was in my clipboard.

**Comment by halfnibble** [comment]
OMG. So it was in my clipboard!

**Comment by devinbaeten** [comment]
It’s quite convenient that you put the code in our clipboard during with, but from a UX standpoint this is very bad.

Writing to the clipboard without a users knowledge could lead to loss of data and or (evidently) confusion. 

Please consider making an alert that displays the code or a prompt asking to save the code to the clipboard.

**Comment by gobijan** [comment]
Feel free to open a PR an I'll review it :)

**Comment by wptechnology** [comment]
Yes same here... I have lost a lot of time to just finally found someone somewhere that says that it should be copied in the clipboard...
Men, if you can't show an alert window, please write something in the extension console at least that says "Device code copied in clipboard... Just have to past it in the GitHub Page" !!!

**Comment by gobijan** [comment]
This was just a quick exploration to see if it was feasible. At the moment I currently do not plan on allocating time to this extension.

Feel free to create a PR with documentation fixes and the alert and log improvements. This shouldn't be too hard. Give it a try :)

**Comment by TheBoatyMcBoatFace** [comment]
Solution documented in https://github.com/gobijan/nova-copilot-lsp/pull/8

**Comment by TheBoatyMcBoatFace** [comment]
> BenjaminHoegh

Have you tried it with the new docs? I hate to say _it worked for _me_, but when I ran through the nvim steps, things were groovy.

[Here are the docs I'm taking about](https://github.com/gobijan/nova-copilot-lsp/tree/f58485d30ce6e8bd52b99aff6c71a87d34a7f3d6)
